shows_app/src/app/components/feature/auth/RegisterForm/RegisterForm.tsx

```diff
@@ -0,0 +1,101 @@
+"use client";
+
+import { ButtonForm } from "@/app/components/shared/Button";
+import { IRegisterArgs } from "@/app/typings/auths";
+import { mutator } from "@/fetchers/mutators";
+import { swrKeys } from "@/fetchers/swrKeys";
+import { AtSignIcon, LockIcon, PhoneIcon } from "@chakra-ui/icons";
+import {
+  chakra,
+  Flex,
+  FormControl,
+  FormErrorMessage,
+  FormLabel,
+  Heading,
+  Input,
+  Text,
+  Link,
+  InputLeftElement,
+  InputGroup
+} from "@chakra-ui/react";
+import { ErrorMessage } from "@hookform/error-message";
+import { useForm } from "react-hook-form";
+import useSWRMutation from "swr/mutation";
+import { PasswordInput } from "../PasswordInput/PasswordInput";
+
+//izdviji formu !!!!!
+export const RegisterForm = () => {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors, isSubmitting }
+  } = useForm<IRegisterArgs>({});
+  const { trigger } = useSWRMutation(swrKeys.register, mutator);
+
+  const onSubmit = async (data: IRegisterArgs) => {
+    console.log(data);
+    await trigger(data);
+  };
+
+  return (
+    <chakra.form
```

Comment:
Could you add a label here that is shown instead of the form after registering succesfully?